File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int num;
    do{
        printf("Height:\n");
        num = get_int();
    }while(num > 23 || num < 0);
    
    for (int i = 1; i <= num; ++i)
    {
    	for (int k = 0; k < num-i ; ++k)
    	{
    		printf(" ");
    	}
    	for (int j = 0; j < i; ++j)
    	{
    		printf("#");
    	}
    	printf("  ");
    	for (int j = 0; j < i; ++j)
    	{
    		printf("#");
    	}
    	printf("\n");
    }
}

File: pset6/mario/mario.py
from cs50 import get_int
while True:
    num = get_int('Height:')
    if (num >= 0 and num <= 23):
        break
x = 1
while (x <= num):
    print(' ' * (num - x), end='')
    print('#' * x, end='')
    print('  ', end='')
    print('#' * x, end='')
    print()
    x += 1